File: app-data/src/main/java/com/zxjdev/smile/data/user/UserEntity.java
package com.zxjdev.smile.data.user;

import com.avos.avoscloud.AVUser;

public class UserEntity extends AVUser {

  public static final transient Creator CREATOR;

  static {
    CREATOR = AVObjectCreator.instance;
  }

  public void setNickName(String name) {
    this.put("nickName", name);
  }

  public String getNickName() {
    return this.getString("nickName");
  }

  public void setAvatar(String url) {
    this.put("avatar", url);
  }

  public String getAvatar() {
    return this.getString("avatar");
  }
}

File: app-data/src/main/java/com/zxjdev/smile/data/user/entity/UserEntity.kt
package com.zxjdev.smile.data.user.entity

import com.avos.avoscloud.AVUser

class UserEntity : AVUser() {

    var nickName: String?
        get() = this.getString("nickName")
        set(name) = this.put("nickName", name)

    var avatar: String?
        get() = this.getString("avatar")
        set(url) = this.put("avatar", url)
}